docs/references/fields.md

```diff
@@ -20,6 +20,33 @@ class Fruit:
     name: str
 ```
 
+# Choice fields
+
+For IntegerChoices Enum can be used to display the value
+
+```python
+# models.py
+class FruitColor(models.IntegerChoices):
+  Red = 1
+  Yellow = 2
+  Green = 3
+
+# types.py
+from enum import Enum
+
+@strawberry.enum
+class FruitColor(Enum):
+  Red = 1
+  Yellow = 2
+  Green = 3
```

Comment:
Since `IntegerChoices` is an enum itself, this can probably be simplified to:

```suggestion
FruitColor = strawberry.enum(models.FruitColor, name="FruitColor")
```

Maybe mention the two cases?

Reply:
Not to mention that the enum itself in the models.py module can be decorated and used directly by both django and strawberry